addOns/postman/src/main/java/org/zaproxy/addon/postman/models/Body.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2023 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.postman.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.List;
+import org.zaproxy.addon.postman.deserializers.ListDeserializer;
+import org.zaproxy.addon.postman.deserializers.ObjectDeserializer;
+
+/** Represents the body of a request in the Postman format. */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Body {
+
+    private static final List<String> ALLOWED_MODES =
+            List.of("raw", "urlencoded", "formdata", "file", "graphql");
+
+    @JsonDeserialize(using = ObjectDeserializer.class)
+    private String mode;
+
+    @JsonDeserialize(using = ObjectDeserializer.class)
+    private String raw;
+
+    @JsonDeserialize(using = ListDeserializer.class)
+    private List<KeyValueData> urlencoded;
+
+    @JsonDeserialize(using = ListDeserializer.class)
+    private List<FormData> formData;
+
+    @JsonDeserialize(using = ObjectDeserializer.class)
+    private File file;
+
+    @JsonDeserialize(using = ObjectDeserializer.class)
+    private GraphQL graphql;
```

Comment:
For the type we use `GraphQl` and for the variable `graphQl`, in other places of the codebase.